pkg/kubelet/cm/dra/manager.go

```diff
@@ -254,3 +255,26 @@ func (m *ManagerImpl) UnprepareResources(pod *v1.Pod) error {
 func (m *ManagerImpl) PodMightNeedToUnprepareResources(UID types.UID) bool {
 	return m.cache.hasPodReference(UID)
 }
+
+// GetPodClaimInfoState gets Pod's ClaimInfoState
+func (m *ManagerImpl) GetPodClaimInfoState(pod *v1.Pod, container *v1.Container) ([]*state.ClaimInfoState, error) {
+	claimInfoState := []*state.ClaimInfoState{}
```

Comment:
nit: initialize with `len(pod.Spec.ResourceClaims)` (just because you do something similar elsewhere in this PR)